docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -211,7 +210,22 @@ If multiple entries are specified in the `host_user_uid` or `host_user_gid` only
 
 </Admonition>
 
-## Step 3/3 Test host user creation
+## Step 3/4 Sudoers Configuration
+
+Teleport host user creation leverages the `sudoers.d` directory for new users. 
+For CentOS builds, ensure that the following line is present in your sudoers file. Please ignore for any other Linux distros:
```

Comment:
```suggestion
For CentOS builds, ensure that the following line is present in your sudoers file. Ignore for any other Linux distros:
```
There's no need for "please" in documentation since the expectation is that documentation tells a reader what to do.